src/main/java/uk/gov/hmcts/reform/hmc/service/LinkedHearingGroupServiceImpl.java

```diff
@@ -172,16 +187,45 @@ private List<HearingResponseEntity> getLatestVersionHearingResponses(HearingEnti
 
     private void deleteFromLinkedGroupDetails(List<HearingEntity> linkedGroupHearings) {
         LinkedGroupDetails linkedGroupDetails = linkedGroupHearings.get(0).getLinkedGroupDetails();
+        final String requestId = linkedGroupDetails.getRequestId();
         saveLinkedGroupDetailsAudit(linkedGroupDetails);
         linkedGroupHearings.forEach(hearingEntity -> saveLinkedHearingDetailsAudit(hearingEntity));
-        saveLinkedGroupDetails(linkedGroupDetails);
-        // TODO: call ListAssist - https://tools.hmcts.net/jira/browse/HMAN-97
+        saveLinkedGroupDetails(linkedGroupDetails, requestId);
+        try {
+            futureHearingRepository.deleteLinkedHearingGroup(requestId);
+            log.info("Response received from ListAssist successfully");
```

Comment:
```suggestion
            log.debug("Response received from ListAssist successfully");
```
To avoid this constantly coming up in Prod - it doesn't really provide any value in normal usage